### PR TITLE
JurassicNinja : use direct betadownload.jetpack.me url i tampermonkey script

### PR DESCRIPTION
**Pull request by mdbitz**
Jurassic.Ninja is migrating to WP.com. Tampermonkey has a dependency on the jetpack-beta/branches endpoint that was a pass through for betadownload.jetpack.me. 

## Proposed changes:

* Update Tampermonkey script to use betadownload.jetpack.me endpoint

### Other information:

- [N/A] Have you written new tests for your changes, if applicable?
- [N/A] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [N/A] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

https://wp.me/p9o2xV-3Fc

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

* Modify your local tampermonkey script to use the version in this PR
* Access a Jetpack pull request
* Verify the tampermonkey script executes and when creating the new site it set the beta branch correctly.

### CODE REVIEW

**Comment by mdbitz** [comment]
Note: I'm not sure why we were making a request to jurassic.ninja to then request from betadownload.jetpack.me which would have increased the time to fulfill the request.

File: tools/jetpack-live-branches/jetpack-live-branches.user.js [hunk 1]
```diff
@@ -147,16 +147,14 @@
...
 						}
-					} else if ( body.code === 'rest_no_route' ) {
+					} else {
 						plugins.push( {
 							name: 'branch',
 							value: currentBranch,
 							label: 'Jetpack',
 							checked: true,
 							disabled: true,
 						} );
```
Comment (anomiex) [inline tools/jetpack-live-branches/jetpack-live-branches.user.js]: The case you kept was for when we first added multiple-branch support. Probably best to keep the error case instead.
```suggestion
					} else {
						throw new Error( 'Invalid response from server' );
```

File: tools/jetpack-live-branches/jetpack-live-branches.user.js [hunk 2]
```diff
@@ -112,19 +112,19 @@
 		} else {
 			if ( ! pluginsList ) {
 				pluginsList = dofetch(
-					`${ host }/wp-json/jurassic.ninja/jetpack-beta/branches/${ repo }/${ currentBranch }`
+					`https://betadownload.jetpack.me/query-branch.php?repo=${ repo }&branch=${ currentBranch }`
```
Comment (anomiex) [inline tools/jetpack-live-branches/jetpack-live-branches.user.js]: Probably my fault these didn't have `encodeURIComponent()` already.
```suggestion
					`https://betadownload.jetpack.me/query-branch.php?repo=${ encodeURIComponent( repo ) }&branch=${ encodeURIComponent( currentBranch ) }`
```

**Review by anomiex — COMMENTED** [review]
Works for me. Left a few suggestions.

File: tools/jetpack-live-branches/jetpack-live-branches.user.js (hunk 2 above)
Comment (mdbitz) [reply tools/jetpack-live-branches/jetpack-live-branches.user.js]: Thanks, added.

**Comment by anomiex** [comment]
> Note: I'm not sure why we were making a request to jurassic.ninja to then request from betadownload.jetpack.me which would have increased the time to fulfill the request.

Looks like it started out using the same API endpoints that the /specialops page used for its own branch fields, which I see you also removed in the Atomic-powered version. I don't see an indication on https://github.com/Automattic/jurassic.ninja/commit/774b535280c5e43e39d84528d71b0320082c34c5 as to why that was done originally.

File: tools/jetpack-live-branches/jetpack-live-branches.user.js (hunk 1 above)
Comment (mdbitz) [reply tools/jetpack-live-branches/jetpack-live-branches.user.js]: Makes sense, I was thinking best to default vs throw an error but this aligns to current functionality.

File: tools/jetpack-live-branches/jetpack-live-branches.user.js [hunk 3]
```diff
@@ -112,19 +112,21 @@
 		} else {
 			if ( ! pluginsList ) {
 				pluginsList = dofetch(
-					`${ host }/wp-json/jurassic.ninja/jetpack-beta/branches/${ repo }/${ currentBranch }`
+					`https://betadownload.jetpack.me/query-branch.php?repo=${ encodeURIComponent(
+						repo
+					) }&branch=${ encodeURIComponent( currentBranch ) }`
```
Comment (anomiex) [inline tools/jetpack-live-branches/jetpack-live-branches.user.js]: I hate when prettier does this, it screws up syntax highlighting in vim. If you hate it too we could do like this
```suggestion
					// prettier-ignore
					`https://betadownload.jetpack.me/query-branch.php?repo=${ encodeURIComponent( repo ) }&branch=${ encodeURIComponent( currentBranch ) }`
```

**Review by anomiex — COMMENTED** [review]
Works for me. One optional inline comment.

**Review by anomiex — DISMISSED** [review]

File: tools/jetpack-live-branches/jetpack-live-branches.user.js (hunk 3 above)
Comment (mdbitz) [reply tools/jetpack-live-branches/jetpack-live-branches.user.js]: Yes, I was just looking for that syntax :)

**Review by anomiex — APPROVED** [review]

**Pull request merged**